Implementación Geolocation

darkukore opened the pull request:
Se agrega la función de geolocation de OpenWeather la cual incluye: País Estado, Ciudad, Logitud y Latitud.

JosuePlataV reviewed (approved):
Basándome en el trabajo que hice utilizando el API de OpenWeather el parcial pasado, parece que las funciones que utilizan Openweather hacen los llamados de los parámetros de manera correcta, los cambios hechos al código previamente existente parecen estar correctos.

TilinMamarre69 reviewed (approved):
Se aprueban los cambios para observar cómo se comportan la API de Geolocation en Pages.

darkukore committed:
Implementación Geolocation

se agrega la función de geolocation de OpenWeather la cual incluye: País Estado y Ciudad.
